packages/runtime-dom/src/nodeOps.ts

```diff
@@ -24,13 +26,9 @@ export const nodeOps = {
 
   createComment: (text: string): Comment => doc.createComment(text),
 
-  setText: (node: Text, text: string) => {
-    node.nodeValue = text
-  },
+  setText: (node: Text, text: string) => patchNodeValue(node, text),
```

Comment:
Could these not be simply 
```suggestion
  setText: patchNodeValue(node, text),
```
As far as I can tell the other properties are the way they are because they'd either need `.bind` or to be called via `doc.createElement` for example to avoid Illegal invocation

Reply:
No, it is a type of function, you we can do like that
`setText: patchNodeValue,`
`setElementText: patchTextContent`

Reply:
Yes, sorry that's what I meant

Reply:
Done